templates/islandora-object-field-thumbnail.tpl.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+/**
+ * @file
+ * Displays an Islandora object field label.
+ *
+ * Available variables:
+ * - $label: The label text.
+ * - $pid: The ID of the object whose label is being displayed.
```

Comment:
`$object` is also an available variable (missing from islandora-object-field-label.tpl.php as well).